docs/health-checks-and-testing.md

```diff
@@ -0,0 +1,30 @@
+# Health Checks and Testing
+
+## Helm Testing
+
+See [Helm Testing](./helm_testing.md).
+
+## Liveness Health Check
+
+When the `--liveness-check` option is used, Chronicle enables liveness
+depth charge checks to ensure system availability. It starts a background
+task that periodically executes a transaction to Chronicle's
+[System Namespace](./namespaces.md#chronicle-system). At each interval,
+the depth charge operation is executed and timed. If the transaction is
+successfully committed, the elapsed time of the round trip is calculated
+and made available as a histogram metric.
+
+The liveness check task installs a [Prometheus](https://prometheus.io/)
+exporter to record metrics. If the transaction is successfully committed,
+the elapsed time of the round trip is calculated and recorded as a
+histogram metric in the Prometheus exporter. The collected data can be
+scraped from the exposed endpoint at `127.0.0.1:9000` for analysis and
```

Comment:
It seems to be listening on every interface when I try it.

Reply:
Thanks. We're at least no longer using the default initialization for the Prometheus builder that uses `0.0.0.0`.